doc/source/schema/schema.md

```diff
@@ -44,26 +59,26 @@ Information and diagnostics for each `posterior` sample, provided by the inferen
 backend. It may vary depending on the algorithm used by the backend (i.e. an affine
 invariant sampler has no energy associated). Therefore none of these parameters
 should be assumed to be present in the `sample_stats` group. The convention
-below serves to ensure that _if_ a variable is present with one of these names
-it will correspond to the definition included here.
+below serves to ensure that if a variable is present with one of these names
+it will correspond to the definition given in front of it.
 
 The name convention used for `sample_stats` variables is the following:
 
 * `lp`: The joint log posterior density for the model (up to an additive constant).
 * `acceptance_rate`: The average acceptance probabilities of all possible samples in the proposed tree.
 * `step_size`: The current integration step size.
-* `step_size_nom`: The nominal integration step size. The `step_size` may differ from this, for example if the step size is jittered. Should only be present if `step_size` is also present and it varies between samples (i.e. step size is jittered).
+* `step_size_nom`: The nominal integration step size. The `step_size` may differ from this for example, if the step size is jittered. It should only be present if `step_size` is also present and it varies between samples (i.e. step size is jittered).
 * `tree_depth`: The number of tree doublings in the balanced binary tree.
 * `n_steps`: The number of leapfrog steps computed. It is related to `tree_depth` with `n_steps <=
   2^tree_dept`.
 * `diverging`: (boolean) Indicates the presence of leapfrog transitions with large energy deviation
   from starting and subsequent termination of the trajectory. "large" is defined as `max_energy_error` going over a threshold.
-* `energy`: The value of the Hamiltonian energy for the accepted proposal (up to an
+* `energy`: The value of the [Hamiltonian energy](https://en.wikipedia.org/wiki/Hamiltonian_(quantum_mechanics)) for the accepted proposal (up to an
```

Comment:
we assumed people reading that are already familiar with MCMC and HMC samplers. Otherwise, the definitions as well as the required introduction on samplers preceding them would be too long. I am therefore hesitant about adding links here because I think that if they don't know what that means they won't learn reading any single link to wikipedia either, maybe a book chapter :thinking: 

If you think the link is needed, it should point to https://en.wikipedia.org/wiki/Hamiltonian_mechanics instead, not to the quantum mechanics meaning.